### PR TITLE
Mount synced `ca-bundle` ConfigMap into kubermatic-seed-controller-manager

### LINKED ISSUE #13937: KKP custom CA bundle not synchronized to the seed properly
### What happened?
After installing custom CA bundle following the guide [in the docs]( https://docs.kubermatic.com/kubermatic/v2.26/tutorials-howtos/kkp-configuration/custom-certificates/) with a name `ca-bundle-new`, the seed has become unhealthy with reconciliation errors:
```
failed to reconcile Kubermatic Deployments: failed to ensure Deployment         kubermatic/kubermatic-seed-controller-manager: failed to build bject(*v1.Deployment) ''kubermatic/kubermatic-seed-controller-manager'': failed to determine revision labels for volumes: could not get ConfigMap kubermatic/ca-bundle-customized: ConfigMap "ca-bundle-new" not found
```

It was verified that the bundle configured on master is synced to the seed under the standard name `ca-bundle` but the deployment of `seed-controller-manager` is looking for `ca-bundle-new` which exists only on the seed.

### Expected behavior
After configuring the custom ca bundle the seed is supposed to reconcile properly.

### How to reproduce the issue?
described above

### How is your environment configured?

- KKP version: 2.26
- Shared or separate master/seed clusters?: separate

### Provide your KKP manifest here (if applicable)

<details>

```yaml
# not relevant ?
```

</details>

### What cloud provider are you running on?
not relevant

### What operating system are you running in your user cluster?
not relevant

### PR DESCRIPTION
**Pull request by embik**
**What this PR does / why we need it**:

It looks like we had a bug in our custom CA handling going all the way back to #6538:

- We allow configuring a custom CA bundle in `KubermaticConfiguration`. This is a `ConfigMap` reference, the default CA bundle we ship is called `ca-bundle`, but the reference doesn't require that name (in fact, our docs recommend a different name).
- This configured CA bundle is synced from master to seeds [here](https://github.com/kubermatic/kubermatic/blob/2827152ad017054e2af357fb5d5d2b118ceb5bb5/pkg/controller/operator/seed/reconciler.go#L503). The reconciler is hardcoded to a constant that uses `ca-bundle` for the `ConfigMap` name it should create on the seed.
- The reconciler for the kubermatic-seed-controller-manager `Deployment` references the `ConfigMap` reference that is configured in the `KubermaticConfiguration` singleton. This works fine if master and seed are on the same cluster (since the source `ConfigMap` exists there), but explodes once master and seed are separate environments (because the sync always creates `ca-bundle` on the seed cluster).

TL;DR: YAMSB (Yet Another Master/Seed Bug).

**Which issue(s) this PR fixes**:

Fixes #13937

**What type of PR is this?**
/kind bug

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
Mount correct `ca-bundle` ConfigMap in kubermatic-seed-controller-manager Deployment on dedicated master/seed environments
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

**Review by xrstf — APPROVED** [review]
/approve
